Khan/perseus
# Clean up + make InteractiveGraphSettings more specific to InteractiveGraphEditor

## Summary:

Minor refactoring of InteractiveGraphSettings.
- Remove `editableSettings` prop because it wasn't being passed in at all by InteractiveGraphEditor, just using defaults.
- Add docstrings to props so they show up in stories.
- Remove string refs.
- Favor destructuring objects over underscore library's clone function
- Remove the `canvas` section entirely since it will never be rendered
- Add `key` prop to map items

Issue: https://khanacademy.atlassian.net/browse/LC-1702

## Test plan:

`yarn jest`

Storybook
- Go to Editor Page story (http://localhost:6006/?path=/docs/perseus-editor-editorpage--docs)
- Add an InteractiveGraph widget
- Try updating every value in the settings and confirm that the graph updates accordingly
